tests/acceptance/search/search-filters-test.ts

```diff
@@ -0,0 +1,62 @@
+import { click as untrackedClick, visit } from '@ember/test-helpers';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { click, setupOSFApplicationTest } from 'ember-osf-web/tests/helpers';
+import { clickTrigger } from 'ember-power-select/test-support/helpers';
+import { module, test } from 'qunit';
+
+
+const moduleName = 'Acceptance | search | filters';
+
+module(moduleName, hooks => {
+    setupOSFApplicationTest(hooks);
+    setupMirage(hooks);
+
+    test('add and remove search filters', async assert => {
+        // Load search page
+        await visit('/search');
+        // assert there are no search filters
+        assert.dom('[data-test-filter-facet]').doesNotExist('No filterable properties shown initially');
+        // assert that mobile only side panel toggle is not shown
+        assert.dom('[data-test-toggle-side-panel]').doesNotExist('Side panel toggle not shown in desktop view');
+        // conduct a search
+        await click('[data-test-search-submit]');
+        // assert there are filterable properties
+        assert.dom('[data-test-filter-facet]').exists({ count: 3 }, 'Filterable properties shown after search');
+        // assert there are no active filters
+        assert.dom('[data-test-active-filter]').doesNotExist('No active filters shown initially');
+        // expand a filterable property
+        await click('[data-test-filter-facet-toggle="License"]');
+        // ensure there are filter options
+        assert.dom('[data-test-filter-facet-value]')
+            .exists({ count: 2 }, 'Filter options shown after expanding a filterable property');
+        // click on a filter option
+        await click('[data-test-filter-facet-value] a');
```

Comment:
So you click a link to activate a filter facet rather than using a checkbox making them buttons or similar? This may be an [accessibility violation](https://www.w3schools.com/accessibility/accessibility_buttons_links.php). You might need to use `Button`s with the 'fake-link' layout.